torch/csrc/jit/tracer.h

```diff
@@ -184,8 +184,8 @@ inline Value* getOutputTrace(const std::shared_ptr<TracingState>& state, const V
 // reference to at::Tensor buffer to call unsafeGetTH, but you can't get this
 // out of a const vector (silly std::vector...)
 inline std::pair<std::shared_ptr<TracingState>, variable_list> enter(
-    variable_list inputs, std::size_t num_stages) {
-  auto state = std::make_shared<TracingState>(num_stages);
+    variable_list inputs, std::size_t num_stages, bool creates_handles) {
```

Comment:
Just look how many places needed to be changed because the signature changed here. Since this is mostly a legacy thing, how about we remove it from there, and do a hacky `tracing_state->create_handles = true;` in those few places that need it.